donate/settings/development.py

```diff
@@ -0,0 +1,28 @@
+from .environment import env
+from .database import Database
+from .base import Base
+from .redis import Redis
+from .braintree import Braintree
+
+
+class Development(Base, Database, Redis, Braintree):
+    DEBUG = env('DEBUG')
+    DJANGO_LOG_LEVEL = env('DJANGO_LOG_LEVEL')
+
+    SECRET_KEY = env('DJANGO_SECRET_KEY')
+    ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+
+    CONTENT_TYPE_NO_SNIFF = env('CONTENT_TYPE_NO_SNIFF')
+    CORS_WHITELIST = env('CORS_WHITELIST')
+    CORS_REGEX_WHITELIST = env('CORS_REGEX_WHITELIST')
+    SSL_REDIRECT = env('SSL_REDIRECT')
+    USE_X_FORWARDED_FOR = env('USE_X_FORWARDED_FOR')
+    X_FRAME_OPTIONS = env('X_FRAME_OPTIONS')
+    XSS_PROTECTION = env('XSS_PROTECTION')
+    CSRF_COOKIE_SECURE = env('CSRF_COOKIE_SECURE')
+    SESSION_COOKIE_SECURE = env('SESSION_COOKIE_SECURE')
+
+    AWS_LOCATION = 'donate'
```

Comment:
You might want to remove this if you don't use S3 for local dev

Reply:
yeah I'll leave this to the S3 configuration mixin